modules/nf-core/samtools/bam2fq/main.nf

```diff
@@ -4,12 +4,12 @@ process SAMTOOLS_BAM2FQ {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/samtools:1.17--h00cdaf9_0' :
-        'biocontainers/samtools:1.17--h00cdaf9_0' }"
+        'https://depot.galaxyproject.org/singularity/samtools:1.19.2--h50ea8bc_0' :
+        'biocontainers/samtools:1.19.2--h50ea8bc_0' }"
 
     input:
     tuple val(meta), path(inputbam)
```

Comment:
Here it would be safer to pass a single tuple as inout containing [meta, inputbam, split] to avoid messing up inputs when multiple samples are processed through the reads_channel. This line would be:

`tuple val(meta), path(inputbam), val(split)`

I'll add a comment on how to build the input channel in the decontamination.nf script to match this

Reply:
I'd better remove that split argument from inputs as Chris suggested